nuxeo-java-client-test/src/test/resources/test-portal-sso-login-contrib.xml

```diff
@@ -0,0 +1,62 @@
+<component name="org.nuxeo.ecm.platform.authenticator.portal.sso.config">
+
+    <require>org.nuxeo.ecm.platform.ui.web.auth.WebEngineConfig</require>
+    <require>org.nuxeo.ecm.platform.login.Portal</require>
+    <require>org.nuxeo.ecm.automation.server.auth.config</require>
+
+    <require>org.nuxeo.ecm.restapi.server.auth.config</require> <!-- specificChains -->
+
+    <extension target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService" point="authenticators">
+        <authenticationPlugin name="PORTAL_AUTH">
+            <loginModulePlugin>Trusting_LM</loginModulePlugin>
+            <parameters>
+                <!-- define here shared secret between the portal and Nuxeo server -->
+                <parameter name="secret">nuxeo5secretkey</parameter>
+                <parameter name="maxAge">3600</parameter>
+            </parameters>
+        </authenticationPlugin>
+    </extension>
+
+    <!-- Include Portal Auth into authentication chain -->
+    <extension target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService" point="chain">
```

Comment:
You don't need to override the main authentication chain here as you are only testing the `automation/login` endpoint here.